src/providers/aws.rs

```diff
@@ -827,6 +880,134 @@ impl RegionLauncher {
         Ok(self)
     }
 
+    #[instrument(level = "trace", skip(self, mk))]
+    async fn make_placement<R>(
```

Comment:
I know this is private but worth documenting briefly since the point of `mk` is a little nonobvious from its type/name.